### PR TITLE
Add 1.28&1.29 to kind e2e presubmits

### PR DESCRIPTION
**Pull request by liangyuanpeng**
Add 1.28&1.29 to kind e2e test.

I just copy write from `pull-kind-e2e-kubernetes-1-27`, so I hope this PR didn't miss anything, please help me review it in detail,Thanks.

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml [hunk 1]
```diff
@@ -235,6 +235,98 @@ presubmits:
...
+        - -c
+        - cd ./../../k8s.io/kubernetes && ./../../sigs.k8s.io/kind/hack/ci/e2e.sh
+        env:
+        - name: FOCUS
+          value: "."
+        # TODO(bentheelder): reduce the skip list further
+        # NOTE: changes should filter down to pull-kubernetes-e2e-kind-canary
+        # and then pull-kubernetes-e2e-kind
+        - name: SKIP
+          value: \[Slow\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|PodSecurityPolicy|LoadBalancer|load.balancer|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing
```
Comment (liangyuanpeng) [inline config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml:316]: Actually, I want to confirm if something needs to be changed here but I don't know how and I'm keen to get some guidance if it needs to be changed,Thanks :)

**Comment by aojea** [comment]
/ok-to-test

@BenTheElder I think we should also drop 1.24 and 1.25 from presubmits

**Comment by liangyuanpeng** [comment]
> I think we should also drop 1.24 and 1.25 from presubmits

Done, deleted it.

**Pull request merged**